.github/workflows/main.yml

```diff
@@ -46,8 +46,10 @@ jobs:
           mamba install --file requirements/test.txt
           pip install -e .
 
-      - name: Use Xvfb Action
-        uses: GabrielBB/xvfb-action@v1
-        with:
-          run: |
-            bash -c "source $HOME/mambaforge/etc/profile.d/conda.sh && conda activate test_env && python -m run_tests.py"
+      - name: Validate diffpy.pdfgui
+        shell: bash -l {0}
+        run: |
+          conda activate test
+          coverage run -m unittest run_tests.py
```

Comment:
this is wrong.  We want to use pytest to run tests.  If you are changing back what I had, please change it back to the cookiecutter version first, and then let's go from there.

Reply:
I tried running pytest on my end, but it will give errors, possible cause by the customized tests for wx. So I'm thinking of making it work with unittest first and see if we can refactor it to pytest